rules/windows/persistence_runtime_run_key_startup_susp_procs.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/11/19"
+maturity = "production"
+updated_date = "2020/11/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of suspicious persistent programs (scripts, rundll32, etc.) by looking at process lineage and
+command line usage.  
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution of Persistent Suspicious Program"
+risk_score = 47
+rule_id = "e7125cea-9fe1-42a5-9a05-b0792cf86f5a"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+/* userinit followed by explorer followed by early child process of explorer (unlikely to be launched interactively) within 1m */
+sequence by host.id, user.name with maxspan = 2m
```

Comment:
I think this is long wide of any interval, and might catch too much.

Reply:
while testing it took sometime 4 to 5m (depend on system startup speed), anyway to be on safe side (noise) will set it to 1m